<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_Layout">
    <item>
     <layout class="QHBoxLayout" name="data_source_Layout">
      <item>
       <widget class="QLabel" name="data_source_labal">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;数据源:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="data_source">
        <item>
         <property name="text">
          <string>01罐</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>02罐</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>03罐</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="time_Layout">
      <item>
       <widget class="QLabel" name="time_from_labal">
        <property name="text">
         <string>时间从:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="s_datetime"/>
      </item>
      <item>
       <widget class="QLabel" name="to_labal">
        <property name="text">
         <string>到:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="e_datetime"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="sql_table_show">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>631</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="tem_limited_Layout">
      <item>
       <widget class="QLabel" name="tem_limit_show">
        <property name="text">
         <string>临界温度设置:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="tem_limit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="tem_limit_update">
      <property name="text">
       <string>临界温度设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>620</y>
     <width>169</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="tem_limit_on_off">
      <property name="text">
       <string>单独查询临界温度以上数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Confirm_button">
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
